source/hardware/controllers/reloop_ready.rst

```diff
@@ -0,0 +1,92 @@
+.. _reloop-ready:
+
+Reloop Ready
+=============
+
+.. sectionauthor::
+   Nikolaus Einhauser <nikolaus.einhauser@mixxx.org>
+
+The Reloop Ready controller is an entry-level 2-Deck :term:`MIDI` controller
+featuring a built-in 4-channel USB audio interface.
+
+-  `Manufacturer’s product page <https://www.reloop.com/reloop-ready>`__
+-  `Serato DJ Hardware Page <https://serato.com/dj/hardware/reloop-ready>`__
+-  `Instruction Manual <https://www.reloop.com/media/catalog/product/pdf/2/4/3/243598_Reloop_IM.pdf>`__
+
+..
+    -  `Mapping Forum Thread <>`__ TODO
+
+.. versionadded:: 2.3.3
+
+
+Audio Setup
+-----------
+
+For the controllers internal Audio interface to interact properly with Mixxx,
+the following channel assignments are required:
+
+===================== ================
+Output Channels       Assigned to
+===================== ================
+1-2                   Main
+3-4                   Headphones
+===================== ================
+
+.. figure:: ../../_static/controllers/reloop_ready_front.svg
+    :align: center
+    :width: 100%
+    :figwidth: 100%
+    :alt: Reloop-Ready (schematic view)
+    :figclass: pretty-figures
+
+    Reloop-Ready (schematic view)
+
+
+========  ==================================================  ==========================================
+No.       Control                                             Function
+========  ==================================================  ==========================================
+1 & 2     :hwlabel:`GAIN`, :hwlabel:`High`, :hwlabel:`LOW`    Gain / EQ, controls the Eq depending on the chosen setting TODO
+3         :hwlabel:`BROWSE` Encoder                           Navigate the library view up and down
+3         :hwlabel:`BROWSE` Encoder Button                    See :mixxx:coref:`[Library],GoToItem`
+3         :hwlabel:`SHIFT` + :hwlabel:`BROWSE` Encoder        Move the library Focus horizontally
```

Comment:
```suggestion
3         :hwlabel:`SHIFT` + :hwlabel:`BROWSE` Encoder        Move the library focus horizontally
```

For my understanding there are a few words capitalized unnecessary. Please verify.